atomic_reactor/inner.py

```diff
@@ -394,6 +391,9 @@ def build_docker_image(self):
                 raise
 
             return self.build_result
+        except Exception as ex:
+            logger.debug("caught exception (%r) so running exit plugins", ex)
```

Comment:
Is this `except` block only for logging the error? (Not against it, just trying to understand it)